### PR TITLE
Make child group wording consistent.

### LINKED ISSUE #20166: Wrong text when creating a sub group
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

admin/ui

### Describe the bug

I'm nit-picking here, but should the text be:
**No sub group in this group** instead of **No groups in this sub group**
And then:
**You haven't created any sub groups in this group.**
And then:
**Create sub group**

image

### Version

21.1.1

### Expected behavior

As explained above

### Actual behavior

As explained above

### How to Reproduce?

Go to create a group, then a sub-group

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by ssilvert**
Fixes #20166

### CODE REVIEW

File: js/apps/admin-ui/src/groups/GroupTable.tsx [hunk 1]
```diff
@@ -220,9 +220,7 @@ export const GroupTable = ({
           <ListEmptyState
             hasIcon={true}
             message={t(`noGroupsInThis${id ? "SubGroup" : "Realm"}`)}
-            instructions={t(
-              `noGroupsInThis${id ? "SubGroup" : "Realm"}Instructions`
-            )}
+            instructions=""
```
Comment (edewit) [inline js/apps/admin-ui/src/groups/GroupTable.tsx:223]:
```suggestion
            instructions={t("noGroupsInThisSubGroup")}
```
I guess you still want this text here, but also a text for when there are no groups:
```
"noGroupsInstructions": "You haven't created any groups. Create a group to get started.",
```

If so then you need to select that one based on id:

```suggestion
            instructions={t(id ? "noGroupsInThisSubGroup") : "noGroupsInstructions")}
```

**Review by edewit — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/groups/GroupTable.tsx (hunk 1 above)
Comment (edewit) [reply js/apps/admin-ui/src/groups/GroupTable.tsx:223]: this way the group empty state is in sync with the others

Comment (ssilvert) [reply js/apps/admin-ui/src/groups/GroupTable.tsx:223]: I left it blank intentionally.  The instructions seem redundant.

Comment (ssilvert) [reply js/apps/admin-ui/src/groups/GroupTable.tsx:223]: > In that case the whole prop can be omitted no?

No, we can't omit the prop.  `ListEmptyState` is used all over the place.  We could make it optional, but that seems like a waste of time right now.

Comment (edewit) [reply js/apps/admin-ui/src/groups/GroupTable.tsx:223]: Right, because we want a consistent UI, you can't just leave it blank.

**Pull request closed without merging**